src/main/shadow/cljs/devtools/api.clj

```diff
@@ -690,4 +695,4 @@
            (swap! results-ref update :out str text))
          })
 
-      @results-ref)))
```

Comment:
I didn't delete this on purpose. I think it was the GitHub code “editor” that did it.